src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java

```diff
@@ -77,6 +78,8 @@
       new SampleAggregator(RcaGraphMetrics.values());
   public static final SampleAggregator RCA_RUNTIME_METRICS_AGGREGATOR =
       new SampleAggregator(RcaRuntimeMetrics.values());
+  public static final SampleAggregator RCA_VERTICES_METRICS_AGGREGATOR =
```

Comment:
We should re-use the graph metrics aggregator. Thoughts?

Reply:
yeah, maybe we should not create a separate aggregator. I was thinking we might add a lot more RCA metrics in the future which will make the  RCA_RUNTIME_METRICS_AGGREGATOR looks quite messy.